soc/arm/renesas_smartbond/da1469x/soc.h

```diff
@@ -13,6 +13,16 @@ extern "C" {
 
 #include <DA1469xAB.h>
 
+int z_smartbond_sleep(void);
+void z_smartbond_wakeup_handler(void);
+void z_smartbond_wakeup(void);
+void z_smartbond_restore_state(void);
+
+#define VDD_LEVEL_0V9		0
+#define VDD_LEVEL_1V2		3
+
+void z_smartbond_vdd_level_set(uint8_t vdd_level);
```

Comment:
same

Reply:
can we just keep code that does *basic* settings to initialize power rails on init? we can full configuration via regulators in separate PR but we need time to get familiar with that API.

Reply:
No if it fits the regulator API. Sometimes things require more effort. :-)